modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -22,83 +22,262 @@ Runtime Fabric enables you to specify the level of severity of the message writt
+.GELF Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Match | Specifies the pattern to match for log tags to be output by this plugin. | Yes |
+| Host | Specifies the IP address or hostname of the target Graylog server. | Yes | 127.0.0.1
+| Port | Specifies the port on which your Graylog GELF input is listening. | Yes | 12201
+| Mode | Specifies the protocol to use (`tls`, `tcp`, or `udp`). | Yes | udp
+| Gelf_Short_Message_Key | Specifies a short descriptive message (This must be set in GELF). | Yes | log
+| Gelf_Timestamp_Key | Specifies your log timestamp (This should be set in GELF). | No | timestamp
+| Gelf_Host_Key | Specifies the key for the value that is used as the name of the host, source, or application that sent the message. (This must be set in GELF). | No | host
+| Gelf_Full_Message_Key | Specifies the key to use as the long message, which can contain a backtrace. (This is optional in GELF.) | No | full_message
+| Gelf_Level_Key | Specifies the key to be used as the log level. Its value must be a standard syslog level (between 0 and 7). | No | level
+| Packet_Size | Specifies the size of packets to be sent if the transport protocol is set to `udp`. | No | 1420
+| Compress | Specifies compression of your UDP packets if the transport protocol is set to `udp`. | No | true
+| Tls | Enables or disables TLS support. | No | Off
+|===
+
+Example Fluent Bit output plugin configuration:
+```
+[OUTPUT]
+    Name                    gelf
+    Match                   kube.*
+    Host                    <your-graylog-server>
+    Port                    12201
+    Mode                    tcp
+    Gelf_Short_Message_Key  data
+```
+
+==== Splunk Configuration Parameters
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Splunk Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Host | Specifies the IP address or hostname of the target Splunk service. | Yes | 127.0.0.1
+| Port | Specifies the TCP port of the target Splunk service. | Yes | 8088
+| Splunk_Token | Specifies the Authentication Token for the HTTP Event Collector interface. | Yes |
+| HTTP_User | Specifies an optional username for basic authentication on the HTTP Event Collector. | No |
+| HTTP_Passwd | Specifies a password for the user defined in `HTTP_User`. | No |
+| Tls | Enables or disables TLS support. | No | Off
+|===
 
-=== Procedure
+Example Fluent Bit output plugin configuration:
+```
+[OUTPUT]
+    Name         splunk
+    Host         127.0.0.1
+    Port         8088
+    Splunk_Token xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx
+    Tls          On
+```
+
+=== Kubernetes Metadata
+
+When forwarding logs to external services through Fluent Bit, additional metadata might be available in some providers. When performing searches, the user can leverage these metadata labels to help narrow down search results and focus on the logs only from containers that matters.
+
+[%header%autowidth.spread,cols="a,a,a"]
```

Comment:
@paigehargrave I'm not sure how to make this table look better. Any suggestions?